docs/general/manage-tokens/staking-and-delegating.md

```diff
@@ -1,63 +1,92 @@
 # Staking and Delegating
 
-Staking and Delegation on the Oasis Network is a wonderful way to hold your ROSE
-tokens.
-Here are a few key resources to get started!
+The Oasis network is a proof-of-stake network. This means that the **voting power
+in the network is determined by the amount of tokens an entity posses**. For
```

Comment:
```suggestion
in the network is determined by the amount of tokens staked to an entity**. For
```
(e.g. possession of tokens does not give you more voting power, tokens need to be staked)